internal/common/version.go

```diff
@@ -13,3 +17,10 @@ func VersionGreaterOrEqual(version1, version2 string) (bool, error) {
 	}
 	return !v1.LessThan(v2), nil
 }
+
+func BaseVersionGreaterOrEqual(version1, version2 string) (bool, error) {
```

Comment:
Can you explain why we need it please?

Reply:
Because it less intuitive to use 4.12.0-0-0, most of the time, we do not care if one pre-release is grater than other (pre-release) version, we care only about the comparison of the base version (most of the times).
Also, Not everyone familiar with that format and this leads to a lot of misses and inconsistencies.